ci/taos/checker-pr-gateway.sh

```diff
@@ -210,18 +210,37 @@ function run_all_checkers(){
     # Note that you must modify the Out-of-PR (OOP) killer because OOP killer depends on synchronous method.
     pushd ./taos/
     ${cmd_list[$i]} $checker_args                                        | tee -a $logfile
+    status+=${PIPESTATUS[0]}
```

Comment:
move this to below line 215?
now line 215 gets status of this line.

Reply:
215 gets the recent background pid, unless we do `${cmd_list[$i]} $checker_args | tee -a $logfile &`, it remains 0.

if it is on the background, line 213 operation is not background so it does no affect line 215